### ISSUE TITLE
ModuleNotFoundError: No module named 'graphviz'

### ISSUE THREAD

**Issue by memathyer** [ENCODE-DCC/croo#11]
After successfully running the ATAC-seq pipeline through the Conda environment, Croo was unable to organize my output as it was unable to find the module 'graphviz'. I assumed this was something automatically installed during the installation of Croo through the installation and activation of the pipeline's Conda environment, but if not, is this something I am supposed to manually install?
Thanks!

(encode-atac-seq-pipeline) [mary.mathyer@login01 atac-seq-pipeline]$ croo atac/ba3ff901-3b83-4498-9f30-951aec6c0c86/metadata.json --out-def-json IVL_n4_output.json --out-dir IVL_n4_output --method {copy}
Traceback (most recent call last):
  File "/home/mary.mathyer/miniconda3/envs/encode-atac-seq-pipeline/bin/croo", line 11, in <module>
    from croo.croo import main
  File "/home/mary.mathyer/miniconda3/envs/encode-atac-seq-pipeline/lib/python3.7/site-packages/croo/croo.py", line 17, in <module>
    from .croo_html_report import CrooHtmlReport
  File "/home/mary.mathyer/miniconda3/envs/encode-atac-seq-pipeline/lib/python3.7/site-packages/croo/croo_html_report.py", line 11, in <module>
    from .croo_html_report_task_graph import CrooHtmlReportTaskGraph
  File "/home/mary.mathyer/miniconda3/envs/encode-atac-seq-pipeline/lib/python3.7/site-packages/croo/croo_html_report_task_graph.py", line 11, in <module>
    from graphviz import (Source, render)
ModuleNotFoundError: No module named 'graphviz'

**Comment by leepc12** [comment]
Just released v0.3.2. Please reinstall Croo.
```
pip unistall croo
pip install croo
```
This will install `graphviz`. But PIP-installing `graphviz` doesn't seem to install graphviz's executable `dot`.

Check Croo's version.
```
croo -v
```

You can also completely disable this task graph feature 
```
croo ... --no-graph
```

**Comment by memathyer** [comment]
Reinstalling Croo worked. Thanks!
